modules/navigation/nav_map.cpp

```diff
@@ -108,10 +123,14 @@ void NavMap::set_link_connection_radius(real_t p_link_connection_radius) {
 	iteration_dirty = true;
 }
 
-gd::PointKey NavMap::get_point_key(const Vector3 &p_pos) const {
-	const int x = static_cast<int>(Math::floor(p_pos.x / merge_rasterizer_cell_size));
-	const int y = static_cast<int>(Math::floor(p_pos.y / merge_rasterizer_cell_height));
-	const int z = static_cast<int>(Math::floor(p_pos.z / merge_rasterizer_cell_size));
+const Vector3 &NavMap::get_merge_rasterizer_cell_size() const {
+	return merge_rasterizer_cell_size;
+}
+
+gd::PointKey NavMap::get_point_key(const Vector3 &p_pos, const Vector3 &p_cell_size) {
```

Comment:
isn't this function duplicated now?

Reply:
indeed ... hm ... the map one was intended to stay like is without being static but for some reason I turned them both into the same.